src/launch.js

```diff
@@ -13,7 +13,7 @@ var serviceUri = urlUtils.getUrlParameter("iss");
 var launchContextId = urlUtils.getUrlParameter("launch");
 
 // Change this to the ID of the client that you registered with the SMART on FHIR authorization server.
-var clientId = "default"; // local client
+var clientId = "7c47a01b-b7d8-41cf-a290-8ed607108e70"; // local client
```

Comment:
Is this always the local clientId? Should this be coming from somewhere else so it is not hard coded?

Reply:
this just a workaround for epic